app/src/main/java/org/eyeseetea/malariacare/data/database/model/SurveyDB.java

```diff
@@ -786,6 +788,13 @@ public void reschedule(Date newScheduledDate, String comment) {
         this.save();
     }
 
+    public SurveyAnsweredRatioDB getSurveyAnsweredRatio() {
```

Comment:
to get SurveyAnsweredRatio we should not use the survey model but GetSurveyAnsweredRatioUseCase